NetflixClone/Classes/Scenes/Home/HomeViewController.swift

```diff
@@ -6,125 +6,192 @@
 //
 
 import UIKit
+import Combine
 
-class HomeViewController: UIViewController, UICollectionViewDataSource {
-    
-    enum Defaults {
-        static let moviesPerPage = 20
-    }
+class HomeViewController: UIViewController {
+    // MARK: Properties
+    typealias Section = HomeViewModel.Section
     
-    // MARK: Outlets
+    private var viewModel: HomeViewModel
+    private var subscribers: [AnyCancellable] = []
+
+    var dataSource: UICollectionViewDiffableDataSource<Section, Movie>! = nil
+    var currentSnapshot: NSDiffableDataSourceSnapshot<Section, Movie>! = nil
```

Comment:
```suggestion
    var dataSource: UICollectionViewDiffableDataSource<Section, Movie>!
    var currentSnapshot: NSDiffableDataSourceSnapshot<Section, Movie>!
```
If you force unwrapping a property, no need to set it to `nil` initially.